src/sentry/deletions/defaults/organizationmember.py

```diff
@@ -0,0 +1,10 @@
+from sentry.models.groupsearchview import GroupSearchView
+
+from ..base import ModelDeletionTask, ModelRelation
+
+
+class OrganizationMemberDeletionTask(ModelDeletionTask):
+    def get_child_relations(self, instance):
+        relations = [ModelRelation(GroupSearchView, {"user_id": instance.user_id})]
```

Comment:
Do we need to send the `organization_id` too? This seems like it would delete all of the user's views, but maybe I'm missing something

Reply:
Yup I think you're right about that, fixing that now. I'll add a check in the tests to double check as well